test/openwpmtest.py

```diff
@@ -40,7 +40,8 @@ def get_test_config(self, data_dir="",
             load_default_params(num_browsers)
         manager_params['data_directory'] = data_dir
         manager_params['log_directory'] = data_dir
-        browser_params[0]['headless'] = True
+        for i in range(num_browsers):
```

Comment:
If you think it makes sense, maybe add an OS check here and exclude this on darwin

Reply:
This is a good idea, but I think we should address it globally. I'll make a new PR that disables it in the browser deployment script.

Reply:
See #388